plausible/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: plausible
+icon: https://svgshare.com/i/y30.svg
+category: developer
+name: Plausible Analytics
+version: "2.1.1"
+tagline: Easy to use and privacy-friendly Google Analytics alternative
+description: >-
+  Plausible is intuitive, lightweight and open source web analytics.
+  No cookies and fully compliant with GDPR, CCPA and PECR.
+  Made and hosted in the EU, powered by European-owned cloud infrastructure.
```

Comment:
@arguser @sharknoon I'm wondering if we should add something to the description about needing to expose Plausible to the internet with something like the Cloudflare Tunnel app. What do you think?